carbon-design-system/carbon-addons-iot-react
# fix(tabletoolbar): add aria-label to search input

Closes #3400 

**Summary**

- Add aria label to search input in table toolbar

**Change List (commits, features, bugs, etc)**

- Add `aria-label="search"` to search input
- Update storyshots

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3635--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--with-searching)
- Inspect search input in the browser tool
- Verify that `aria-label` is set for the input field

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
